src/cascadia/TerminalControl/IControlSettings.idl

```diff
@@ -23,7 +24,7 @@ namespace Microsoft.Terminal.TerminalControl
     //      TermControl's behavior. In these settings there is both the entirety
     //      of the Core ITerminalSettings interface, and any additional settings
     //      for specifically the control.
-    interface IControlSettings requires ICoreSettings
+    interface IControlSettings requires ICoreSettings, IControlAppearance
     {

```

Comment:
This might be in the spec, but why aren't we including...
- `FontFace`
- `FontSize`
- `FontWeight`
- retro terminal effects
- ~acrylic~ oh duh, we can't
- ~acrylic opacity~ oh duh, we can't
- Padding
- Scrollbar visibility
They sound like they should be appearance settings. I just went down the line of settings in the Profiles > Appearance page in the settings UI.

Reply:
On a related note, (before or after you address the comment above), what happens if you do the following:
1. ctrl+scroll to change the font size at runtime
2. lose/gain focus (apply appearance settings)

Does the user lose their runtime font size? I'm not sure what the correct behavior should be here but we should probably think about it.

Reply:
The user does not lose their runtime font size - that was basically the purpose of #8602, control overrides go in the child and settings updates (either through reload or in this case, focus change) go in the parent, so the child ones take precedence

Reply:
We don't have `Padding`/`FontFace`/`Fontsize` because I believe they cause a resize (based off the description in [#3062](https://github.com/microsoft/terminal/issues/3062)). Regarding the others, we decided that we will wait to see how this performs and they can easily be added in later (because there is some dubiousness in whether certain other settings are considered 'appearance', like is bellstyle: visual an appearance setting? We thought it best to get the foundation for unfocused configurations down and then come back to it with other things we want to add)

Reply:
Shader path and retro effects have been moved to appearance!